third_party/Makefile

```diff
@@ -116,20 +116,35 @@ gpcnet: sb_micro_path
 	cp -v ./GPCNET/network_test $(SB_MICRO_PATH)/bin/
 	cp -v ./GPCNET/network_load_test $(SB_MICRO_PATH)/bin/
 
-#Build GPU burn from main branch (only branch that exists)
+# Build GPU burn from main branch (only branch that exists)
 cuda_gpuburn: sb_micro_path
 ifneq (,$(wildcard gpu-burn/Makefile))
 	cd ./gpu-burn && make
 	cp -v ./gpu-burn/gpu_burn $(SB_MICRO_PATH)/bin/
 	cp -v ./gpu-burn/compare.ptx $(SB_MICRO_PATH)/bin/
 endif
 
-#Build stream from main branch (only branch that exists)
+# Build HPL from main branch 
+cpu_hpl: sb_micro_path
+ifneq (,$(wildcard hpl-tests/Makefile))
+	cd ./hpl-tests && \
+    wget https://netlib.org/benchmark/hpl/hpl-2.3.tar.gz && \
+	tar xzf hpl-2.3.tar.gz && \
+	cp Make.Linux_zen3 hpl-2.3 && \
+	cp Make.Linux_zen4 hpl-2.3 && \
+	make all
+	cp -v ./hpl-tests/hpl-2.3/bin/Linux_zen3/xhpl $(SB_MICRO_PATH)/bin/xhpl_z3
+	cp -v ./hpl-tests/hpl-2.3/bin/Linux_zen4/xhpl $(SB_MICRO_PATH)/bin/xhpl_z4
+	cp -v ./hpl-tests/hpl_run.sh $(SB_MICRO_PATH)/bin/
+	cp -v ./hpl-tests/bindmem.sh $(SB_MICRO_PATH)/bin/
+	cp -v ./hpl-tests/template_hpl.dat $(SB_MICRO_PATH)/bin/
```

Comment:
Do we need to keep these 5 files? It would be a big issue on license if we would like to add third-party code directly.

Reply:
I wrote the make files. The make files and the rest of the files are now created by MSFT. Since they are used to build and run HPL can we still keep them in the Third-party directory? Unfortunately, The HPL source code does not come in a git repo. It's found here: https://netlib.org/benchmark/hpl/

Reply:
I see Linux_zen3 and Linux_zen4 are not our code. Others should be OK.

Reply:
I have addressed this by copying and modifying a template via makefile from the HPL source directory.